packages/interface/src/components/preferences-modal-tabs/style.scss

```diff
@@ -1,24 +1,24 @@
 $vertical-tabs-width: 160px;
 
 .interface-preferences__tabs {
-	.components-tab-panel__tabs {
+	.interface-preferences__tabs-tablist {
```

Comment:
Now that we have explicit class names, we can get rid of the unnecessary nesting. (The extra specificity becomes a hassle for anyone that may want to override styles.)

Reply:
Thanks @mirka! Good point on the nesting. I removed it, but I did end up having to modify one of the selectors for the actual tabs. Without the nesting that was in place previously, the `:focus:not(:disabled)` selector wasn't specific enough to take precedence over the component's base style.

I went with adding a `[role="tab"]` selector to boost the specificity here, but I'm open to alternatives if you have any. We could also restore a single level of nesting, and keep all of the `tab` styles nested under the `tablist` style.

Reply:
I like how you did it, it's very clean ✨ Let's merge!

Reply:
Great, thanks for the final thumbs up! merged!